src/GazeboThruster.cpp

```diff
@@ -192,6 +192,7 @@ void GazeboThruster::updateBegin(common::UpdateInfo const& info)
             thruster != thrusters.end(); ++thruster)
     {
         physics::LinkPtr link = model->GetLink( thruster->name );
+        link->AddLinkForce( thruster->effort*gazebo::math::Vector3::UnitX);
         link->AddLinkForce( thruster->effort*thruster->direction, thruster->position );
```

Comment:
Yeah, 
Before I was sending the matrix M(M+Ma)^-1 and editing the original force.
But if Ma=0, the position where the force is applied was no longer (0,0,0), once the position is choose in order that (position-cog) and direction force are [perpendicular](https://github.com/Brazilian-Institute-of-Robotics/simulation-gazebo_thruster/blob/98152f96593773e2c4b3e84361300c9fe8c1cdd0/src/GazeboThruster.cpp#L212). 

So now I'd prefer to keep the original force, and to use the effort edited by the matrix (M(M+Ma)^-1 - I) to apply the compensated force. In case Ma=0, this extra force is also 0, and the original force is not edited.